Makefile

```diff
@@ -0,0 +1,53 @@
+artifact_name       := company-accounts.api.ch.gov.uk
+commit              := $(shell git rev-parse --short HEAD)
+tag                 := $(shell git tag -l 'v*-rc*' --points-at HEAD)
+version             := $(shell if [[ -n "$(tag)" ]]; then echo $(tag) | sed 's/^v//'; else echo $(commit); fi)
+artifactory_publish := $(shell if [[ -n "$(tag)" ]]; then echo release; else echo dev; fi)
+
+.PHONY: all
+all: build
+
+.PHONY: submodules
+submodules: api-enumerations/.git
+
+.PHONY: api-enumerations/.git
+	git submodule init
+	git submodule update
+
+.PHONY: clean
+clean:
+	mvn clean
+	rm -f ./$(artifact_name).jar
+	rm -f ./$(artifact_name)-*.zip
+	rm -rf ./build-*
+
+.PHONY: build
+build: submodules
+	mvn versions:set -DnewVersion=$(version) -DgenerateBackupPoms=false
+	mvn package -DskipTests=true
+	cp ./target/$(artifact_name)-$(version).jar ./$(artifact_name).jar
+
+.PHONY: test
+test: test-unit
+
+.PHONY: test-unit
+test-unit: clean
+	mvn test
+
+.PHONY: package
+package:
+	@test -s ./$(artifact_name).jar || { echo "ERROR: Service JAR not found"; exit 1; }
```

Comment:
Can you echo $(artifact_name) in the error? It will be more helpful when debugging.

Reply:
Done